src/lib.rs

```diff
@@ -10,7 +12,11 @@
 /// println!("{}", shortened);
 /// // => "/h/.s/path"
 /// ```
-pub fn tico(tico: &str) -> String {
+pub fn tico(path: &str) -> String {
+    let home_dir = dirs::home_dir().unwrap();
+    let home_dir_str = home_dir.to_str().unwrap();
```

Comment:
Do we need both `unwrap`s? Can `#to_str` be called on `dirs::home_dir()` directly?

Reply:
It cannot, `dirs::home_dir()` returns an `Option`. I am sure this can be improved tho.